cvmfs/cvmfs_config

```diff
@@ -352,8 +356,6 @@ cvmfs_chksetup() {
     fi
     # Check that /etc/auto.cvmfs is referenced in auto.master
     # Check that /etc/auto.cvmfs is executable on Linux
-    local global_mount_dir
-    global_mount_dir=$CVMFS_MOUNT_DIR
```

Comment:
It's copied a few lines to the top.
global_mount_dir was only set if running on Linux, otherwise it remains empty, which causes a failure when being evaluated in the final condition.